test/cli/require-qunit-test.js

```diff
@@ -0,0 +1,59 @@
+const proxyquire = require( "proxyquire" );
+const resolveStub = {
+	sync( path ) {
+		return path;
+	}
+};
+
+QUnit.module( "requireQUnit", function() {
+	QUnit.test( "finds QUnit in the current working directory", function( assert ) {
+		const localQUnit = {
```

Comment:
Is this worth extracting to a module-scope variable since it has the same value? (In theory, any variations in individual tests that might come up later could then be handled with `Object.assign()` or similar.)